app/antedecorators/gasless.go

```diff
@@ -0,0 +1,169 @@
+package antedecorators
+
+import (
+	"bytes"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	dextypes "github.com/sei-protocol/sei-chain/x/dex/types"
+	oraclekeeper "github.com/sei-protocol/sei-chain/x/oracle/keeper"
+	oracletypes "github.com/sei-protocol/sei-chain/x/oracle/types"
+)
+
+type GaslessDecorator struct {
+	wrapped      sdk.AnteDecorator
```

Comment:
we might want to make this a list, since it's possible that some future cosmos version adds a new builtin ante handler that we want to skip for gasless transactions